### ISSUE TITLE
[question] export best technique

### ISSUE THREAD

**Issue by neel04** [mljar/mljar-supervised#363]
Hi, thanks a lot for the easy-to-use repo @pplonski !!
I had a quick question - can we use the best model `MLJar` creates (say, KNN) then can we get it to output all the exact parameters it used to get that performance?

So that we can use those parameters in `scikit-learn` and use the exact model to reproduce the performance/accuracy achieved by Jar?

**Comment by pplonski** [comment]
@neel04 of course! When you fit the AutoML there is created a directory with all results and details about models. You can pass the directory name in the `results_path` argument, if you don't pass, then the default will be generated, something like `AutoML_1` (`AutoML_*` where `*` is the number with empty dir).

When you navigate into the `result_path` directory you will see directories with all trained models. Each model has a `README.md` file, in this file, you have information about all set hyperparameters.

Here you have an example with `README.md` for Xgboost algorithm: https://github.com/mljar/mljar-examples/blob/master/House_price_regression/AutoML_1/5_Default_Xgboost/README.md

**Comment by neel04** [comment]
ahh, I am an idiot :sweat_smile: to be honest, I have never seen an AutoML product do such comprehensive tasks or even generate a fully-fledged report! half of the ones there (like `autosklearn`) are riddled with bugs. Really glad to find this repo here, thank you for your support and good luck! :+1: (And seriously man, get some sleep you always seem online all the time pushing new updates and closing issues)

**Comment by pplonski** [comment]
;) do you get good ML models with MLJAR?
